deps.windows/60-python.ps1

```diff
@@ -10,30 +10,30 @@ param(
 )
 
 function Enable-PyEnv {
-    $Env:PYENV = "$(Get-Location | Convert-Path)\pyenv-win"
-    $Env:PYENV_ROOT = $Env:PYENV
-    $Env:PYENV_HOME = $Env:PYENV
+    $env:PYENV = "$(Get-Location | Convert-Path)\pyenv-win"
+    $env:PYENV_ROOT = $env:PYENV
+    $env:PYENV_HOME = $env:PYENV
 
-    $Env:OriginalPath = $Env:Path
+    $env:OriginalPath = $env:Path
 
-    $PathElements = ([Collections.Generic.HashSet[string]]::new([string[]]($Env:Path -split [System.IO.Path]::PathSeparator), [StringComparer]::OrdinalIgnoreCase))
+    $PathElements = ([Collections.Generic.HashSet[string]]::new([string[]]($env:Path -split [System.IO.Path]::PathSeparator), [StringComparer]::OrdinalIgnoreCase))
 
     $PyEnvPaths = @(
-        "${Env:PYENV}\bin"
-        "${Env:PYENV}\shims"
+        "${env:PYENV}\bin"
+        "${env:PYENV}\shims"
     )
-    $Env:Path = ($PyEnvPaths + $PathElements) -join [System.IO.Path]::PathSeparator
+    $env:Path = ($PyEnvPaths + $PathElements) -join [System.IO.Path]::PathSeparator
```

Comment:
Why the capitalization change here?

Reply:
Mainly that `env` (like `script` or other scopes) are not capitalised in Microsoft's documentation, and it also makes it aesthetically easier to read (as the _actually_ capitalised variable name stands out better).

Reply:
Discussed off-thread. Microsoft's documentation itself is inconsistent with capitalization.

While I'd prefer that style/consistency changes be done in a single sweep in their own PR, I don't want to hold this up with trying to excise and separate these changes.